modules/react/checkbox/stories/Checkbox.stories.mdx

```diff
@@ -104,11 +103,9 @@ or `Checkbox.ErrorType.Error` if Form Field is not being used.
 
 <ExampleCodeBlock code={Error} />
 
-## Props
+## Component API
 
-Undocumented props are spread to the underlying `<input type="checkbox">` element.
-
-<ArgsTable of={Checkbox} />
+<SymbolDoc name="Checkbox" hideDescription />
```

Comment:
It is probably intentional to have this at least for now, but just in case, I wanted to point out that there are types like `Checkbox.ErrorType` showing up with their own tables. I feel like this is unnecessary and the same info appears for the `error` prop for `Checkbox`. I think i saw this elsewhere on other components too.
<img width="1037" alt="Screenshot 2023-02-26 at 8 59 18 PM" src="https://user-images.githubusercontent.com/1075861/221471323-a49845c5-542d-498d-89fc-799cf11ab9dd.png">

Reply:
We are deprecating our use of enums and newer components are instead using string unions. I treat enums as objects which is why they render like this. `subComponents` is a misnomer because it is actually any static value. I don't have a problem with the enums rendering like this.

Reply:
Sounds good.